0004/README.md

```diff
@@ -165,8 +165,11 @@ The NATS message payload is an Avro object with the following schema ([0004-exte
         },
         {
             "name":"appVersionName",
-            "type":"string",
-            "doc":"Application version name the data is sent for"
+            "type":[
+                "string",
+                "null"
+            ],
+            "doc":"Application version name the data is sent for. Null in case of the extension does not know about application version endpoint belongs to."
```

Comment:
s/in case of the extension/in case the extension
s/endpoint belongs to/the endpoint belongs to

Reply:
Thanks, updated.